content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -160,7 +160,7 @@ pods that shouldn't be running. A few of the use cases are
 a particular set of users, you can add a taint to those nodes (say,
 `kubectl taint nodes nodename dedicated=groupName:NoSchedule`) and then add a corresponding
 toleration to their pods (this would be done most easily by writing a custom
-[admission controller](/docs/admin/admission-controllers/)).
+[admission controller](/docs/reference/access-authn-authz/controlling-access/)).
```

Comment:
Is this and the next one correct? Why are we changing the link to the admission controllers?

Reply:
I figured updating them to what they are now instead of relying on the redirect was part of the goal for #9286. If not, this PR can just be closed.

Reply:
I think updating the links is a good thing and we should do it, but some of the links in your PR are changed differently than the rest. Most of the links end in `admission-controllers/` which makes sense, but there are a few that end in `controlling-access/`. This one is one of those. I am asking if this change is intended.

Reply:
Oooh >_< I know exactly what happened, I cherry picked the wrong commit. Thanks for catching that, I'll get that taken care of ASAP.

Reply:
Things should be fixed now..I'm double checking all my other branches to make sure I didn't make a similar mistake.